AdornUI-documentation/card.html

```diff
@@ -0,0 +1,168 @@
+                            <button class="btn btn-primary add-to-cart">ADD TO CART</button>
+                        </div>
+                        <div class="overlay flex-hz"><span class="text-overlay">OUT OF STOCK</span></div>
+                    </div>
+                </div>
+                <iframe src="https://www.thiscodeworks.com/embed/61fcd9bbb57bf800152b721e" frameborder="0"></iframe>
+                <h2 class="component-subtitle">Cards with Text Only</h2>
+                <div class="component-container-hz">
+                    <div class="card__container">
+                        <h2 class="card__title">Card Title</h2>
+                        <h3 class="card__subtitle">Card subtitle</h3>
+                        <p class="card__description">Lorem Ipsum is simply dummy text of the printing and typesetting
+                            industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when
+                            an unknown printer took a galley of type and scrambled it to make a type specimen book.</p>
+                    </div>
+                </div>
+                <iframe src="https://www.thiscodeworks.com/embed/61fcdb7fb57bf800152b7221" frameborder="0"></iframe>
+                <h2 class="component-subtitle">Cards with Dismiss Button</h2>
+                <div class="component-container-hz">
+                    <div class="card__container card__container-close">
+                        <div class="card-row flex-hz">
+                            <h2 class="card__title">Card Title</h2>
+                            <i class="fa fa-close close-card-btn"></i>
+                        </div>
+                        <h3 class="card__subtitle">Card subtitle</h3>
+                        <p class="card__description">Lorem Ipsum is simply dummy text of the printing and typesetting
+                            industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when
+                            an unknown printer took a galley of type and scrambled it to make a type specimen book.</p>
+                    </div>
+                </div>
+                <iframe src="https://www.thiscodeworks.com/embed/61fcdbcbb57bf800152b7222" frameborder="0"></iframe>
+            </section>
+        </main>
+        <footer class="footer">
+            <a class="social-media-link" href="https://github.com/shrushti2000"> <i class="fa fa-github"></i></a>
+            <a class="social-media-link" href="https://twitter.com/Shrushti2000"> <i class="fa fa-twitter"></i></a>
+            <a class="social-media-link" href="https://www.linkedin.com/in/shrushti-polekar-8255471a6"> <i
+                    class="fa fa-linkedin"></i></a>
+        </footer>
+    </div>
+    </div>
+</body>
+<script src="../sidebar.js"></script>
+<script src="../card.js"></script>
+<script src="../card.js"></script>
```

Comment:
Imported twice.

Also, this import must be mentioned in the docs as well if you want the user to use functionalities like close button.